ESCPOS.sln

```diff
@@ -3,9 +3,14 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
 VisualStudioVersion = 16.0.29215.179
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ESCPOS.csproj", "ESCPOS\ESCPOS.csproj", "{69B133DE-222B-4061-BC13-CE345CF176FD}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ESCPOS", "ESCPOS\ESCPOS.csproj", "{69B133DE-222B-4061-BC13-CE345CF176FD}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ESCPOSTest", "ESCPOSTest\ESCPOSTest.csproj", "{A4759F20-0E7D-41A5-B0C0-239FED8D9383}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ESCPOSTest", "ESCPOSTest\ESCPOSTest.csproj", "{A4759F20-0E7D-41A5-B0C0-239FED8D9383}"
```

Comment:
Apparently you used the same GUID of the other project (ESCPOS.csproj)

I don't think that was intended so I am lefting this out.